.github/workflows/copilot_deploy.yml

```diff
@@ -61,7 +59,16 @@ jobs:
           enable-cache: true
       - name: install dependencies
         run: uv sync
+      - name: build static assets
+        env:
+          FLASK_ENV: "development"
+        run: uv run python build.py
+      - name: run unit tests
+        env:
+          GOV_NOTIFY_API_KEY: ${{ secrets.GOV_NOTIFY_API_KEY }}
+        run: uv run pytest -n 2 .
       - name: run type checking
+        if: ${{ needs.setup.outputs.jobs_to_run == '[]' }}
         run: uv run mypy
```

Comment:
This will add a bit of time to the test run as this is now running sequentially rather than in parallel. Do we want to split this out?

Reply:
Made all the checks run in parallel